WordPress/Classes/ViewRelated/EEUUSCompliance/ComplianceLocationService.swift

```diff
@@ -0,0 +1,14 @@
+import WordPressKit
+
+final class ComplianceLocationService {
+    func getIPCountryCode(completion: @escaping (Result<String, Error>) -> Void) {
+        IPLocationRemote().fetchIPCountryCode { result in
+            switch result {
+            case .success(let countryCode):
+                completion(.success(countryCode))
+            case .failure(let error):
+                completion(.failure(error))
+            }
+        }
+    }
```

Comment:
```suggestion
    func getIPCountryCode(completion: @escaping (Result<String, Error>) -> Void) {
        IPLocationRemote().fetchIPCountryCode(completion: completion)
    }
```

Reply:
I don't think we need to unpack the `Result` type if the method signatures match.